administrator/components/com_templates/src/Model/TemplateModel.php

```diff
@@ -306,13 +306,14 @@ public function prepareCoreFiles($dir, $element, $template)
      * @param   array    $value  The file name.
      * @param   integer  $exid   The template extension id.
      *
-     * @return  integer  Number of files changed.
+     * @return  bool|\RuntimeException  Update successfull or exception object
      *
      * @since   4.0.0
      */
     public function publish($ids, $value, $exid)
     {
         $db = $this->getDatabase();
+        $result = false;
```

Comment:
```suggestion
        $db     = $this->getDatabase();
        $result = false;
```
Fix PHPCS failing in Drone.